unity-ggjj/Assets/Scripts/TextDecoder/Preloading/ObjectStorage.cs

```diff
@@ -35,7 +35,14 @@ public void Add(Object obj)
     /// <returns>An object with name objectName</returns>
     public T GetObject<T>(string objectName) where T : class
     {
-        return _objects[objectName] as T;
+        try
+        {
+            return _objects[objectName] as T;
+        }
+        catch (KeyNotFoundException exception)
+        {
+            throw new KeyNotFoundException(objectName);
```

Comment:
Nit: [See earlier comment](https://github.com/Studio-Lovelies/GG-JointJustice-Unity/pull/241#discussion_r838903904).
```suggestion
            throw new KeyNotFoundException(objectName, exception);
```